exercises/G-numbers/exercise.js

```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var message = 'Number of students: ' + numberOfStudents + '\nNumber of mentors: ' + numberOfMentors + '\nTotal number of students and mentors: ' + total;
```

Comment:
Hey Rustumb! 

I am happy to see your efficiency in line 5. I made my code redundant by adding another variable for the number of mentors. Learnt something!